### PR TITLE
Fix #37: Allow LAN access with Kill Switch enabled

### LINKED ISSUE #37: [Enhancement] Kill Switch should allow local network traffic
Currently the kill switch feature prevents all traffic on the local network.

It would be great to have the option to allow traffic on the local network. Or at least, as an advanced option, allow to configure the iptables rules that it uses for the kill switch.

### PR DESCRIPTION
**Pull request by Rafficer**
This allows a computer to still be reachable (and reach other computers) over the local network.

Before merging this, please thoroughly check if you (or someone else on the team) see anything wrong with how it's done and if this can lead to potential leaks in cases the Kill Switch should protect the traffic.

### CODE REVIEW

File: protonvpn_cli/connection.py [hunk 1]
```diff
@@ -815,6 +817,8 @@ def manage_killswitch(mode, proto=None, port=None):
             "iptables -P FORWARD DROP",
             "iptables -A OUTPUT -o lo -j ACCEPT",
             "iptables -A INPUT -i lo -j ACCEPT",
+            "iptables -A OUTPUT -o {0} -s {1} -j ACCEPT".format(default_nic, local_network), # noqa
```
Comment (Rafficer) [inline protonvpn_cli/connection.py]: Should be `-d {1}`

**Comment by Rafficer** [comment]
It just came to my mind that it's maybe wiser to leave this off by default, but make it optional. For example, when on a public wifi (airport, starbucks, etc) it would be better to not allow connections from/to the LAN network. When at home, this is totally fine.

So yeah, please deter the merge until after I get this implemented, which is probably this weekend :)

**Comment by Rafficer** [comment]
Updated now so it's configurable.

**Comment by sargue** [comment]
Looks good, nice solution.

**Pull request merged**